defaults/main.yml

```diff
@@ -48,3 +48,15 @@ rhel7stig_boot_password_config:
     - regexp: ^password_pbkdf2 root
       line: password_pbkdf2 root {{ rhel7stig_bootloader_password | grub2_hash(salt='KeokpkECTJeoDhEA5XtiLQ') }}
 
+rhel7stig_dns_servers:
+  - 8.8.8.8
```

Comment:
Please indicate what vuln these variables are for and brief description of their puprpose.

Please indent four spaces in accordance with the [guidelines](https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md).